### ISSUE TITLE
INavigationAware not invoked on absolute uri navigation

### ISSUE THREAD

**Issue by mikegottlieb** [PrismLibrary/Prism#1021]
### Package info

- Platform: Xamarin.Forms
- Prism version: 6.3
- Xamarin version (if applicable): 2.3.3.193
- Windows 10 SDK version (if applicable): N/A
- Other version info: Android 6.0 emulator

### Repro steps

1. Create a view and viewmodel and register them
2. Have the viewmodel implement INavigationAware
3. Navigate to the page via an absolute uri

OnNavigatingTo and OnNavigatedTo are not invoked.

The scenario is that I have a page that once loaded checks a condition and possibly navigates to another page as a modal. I was originally using view appearance as the hook, but that is super flaky and was causing the modal to get added multiple times. I figured OnNavigatedTo would be the way to go, but it is never invoked.

**Comment by brianlagunas** [comment]
Can you provide a sample that reproduces the issue?  I am unable to reproduce it in my sample.

**Comment by stefffdev** [comment]
Have you tried to clean and rebuild your Android project?
I encountered a similar problem after updating to version 6.3, a rebuild fixed it.

**Comment by brianlagunas** [comment]
I am assuming you fixed this as I have not seen a reply in about a week.  Let me know if you are still having issues.

**Comment by mikegottlieb** [comment]
Sorry, I worked around the issue and moved on to meet a deadline. I just came back to take a second look and this was probably just a mixup on my end.

I am navigating to a TabbedPage/MasterDetailPage via absolute URL. The navigation events on the TabbedPage's ViewModel are firing as expected. The navigation events on the first page inside TabbedPage are not firing. However, since I did not explicitly navigate to that first tab I suppose that is to be expected right?
